### PR TITLE
Update agent docs for v3.43.0

### PR DESCRIPTION
**Pull request by DrJosh9000**

### CODE REVIEW

File: pages/agent/v3/help/_bootstrap.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ See https://buildkite.com/docs/agent/v3/hooks for more details.
...
-<tr id="git-submodule-clone-config"><th><code>--git-submodule-clone-config value</code> <a class="Docs__attribute__link" href="#git-submodule-clone-config">#</a></th><td><p>Comma separated key=value git config pairs applied before git submodule clone commands, e.g. `update --init`. If the config is needed to be applied to all git commands, supply it in a global git config file for the system that the agent runs in instead.<br /><strong>Environment variable</strong>: <code>$BUILDKITE_GIT_SUBMODULE_CLONE_CONFIG</code></p></td></tr>
+<tr id="git-submodule-clone-config"><th><code>--git-submodule-clone-config update</code> <a class="Docs__attribute__link" href="#git-submodule-clone-config">#</a></th><td><p>Comma separated key=value git config pairs applied before git submodule clone commands, e.g. update --init. If the config is needed to be applied to all git commands, supply it in a global git config file for the system that the agent runs in instead.<br /><strong>Environment variable</strong>: <code>$BUILDKITE_GIT_SUBMODULE_CLONE_CONFIG</code></p></td></tr>
```
Comment (triarius) [inline pages/agent/v3/help/_bootstrap.md:67]: 😅

**Review by triarius — APPROVED** [review]
LGTM. Thanks for remembering this!

**Review by mbelton-buildkite — APPROVED** [review]
Looks good to me. I've left one suggestion for you to consider.

I'll also push a change to fix the linting error.

File: pages/agent/v3/cli_env.md.erb [hunk 2]
```diff
@@ -1,12 +1,12 @@
 # `buildkite-agent env`
 
-`env` prints the environment as JSON.
+The Buildkite Agent's `env` subcommands provide the ability to inspect environment variables.
 
 {:toc}
 
 ## Running env
```
[the remark this answers is not in the document]
Comment (DrJosh9000) [reply pages/agent/v3/cli_env.md.erb:7]: Sounds good to me.

**Pull request merged**